### ISSUE TITLE
Links in SECURITY.MD are not accessible - MAS 2.4.4

### ISSUE THREAD

**Issue by HamletDRC** [microsoft/microsoft.github.io#192]
The [Microsoft Accessibility Standards](https://microsoft.sharepoint.com/teams/msenable/Pages/AccessibilityStandard.aspx) (MAS) contains the rule [MAS 2.4.4 – Link Purpose (In Context)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%202.4.4%20%E2%80%93%20Link%20Purpose%20(In%20Context).docx?d=w06a891af210c4c6dbb3e92babab209e7&csf=1&e=jj4etR)

> The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, except where the purpose of the link would be ambiguous to users in general.

There are two ambiguous links on the SECURITY.MD page: "definition" and "many more". Screen reader users who browse the page by scanning the list of links will not know where the "definition" and "many more" links lead. 

I propose to replace the link text "definition" with the text "Microsoft's definition of a security vulnerability" and the text "many more" with the text "our GitHub organizations". Then you would be MAS C compliant, which is the accessibility baseline for many Microsoft products (all of C+AI, for example)

For those without access to the MAS standards, you can also reference [WCAG: Understanding Success Criterion 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

**Comment by jeffwilcox** [comment]
Thanks for the report @HamletDRC and special thanks to @vips28 for providing this fix today. This will be applied to the default templates for all new repos starting today.
